src/dmd/ctorflow.d

```diff
@@ -29,6 +29,7 @@ enum CSX : ubyte
     return_         = 0x20,     /// seen a return statement
     any_ctor        = 0x40,     /// either this() or super() was called
     halt            = 0x80,     /// assert(0)
+    deprecate_18719 = 0x100,    // issue deprecation for Issue 18719 - delete when deprecation period is over
```

Comment:
Can we call it `deprecate` so it can be used for other deprecations?

Reply:
It is highly unlikely that this is going to be used for something else. even so, the modification can be done when it is necessary

Reply:
https://github.com/dlang/dmd/pull/8143#issuecomment-379533909